znai-reactjs/src/doc-elements/structure/tocItem.js

```diff
@@ -14,14 +14,7 @@
  * limitations under the License.
  */
 
-.preview-change-indicator {
-    animation-name: preview-change-keyframes;
-    animation-duration: 2s;
-    animation-fill-mode: both;
-}
-
-@keyframes preview-change-keyframes {
-    0% {background-color: rgba(38, 110, 164, 0)}
-    50% {background-color: rgba(38, 110, 164, 0.5)}
-    100% {background-color: rgba(38, 110, 164, 0)}
+export function areTocItemEquals(a, b) {
```

Comment:
this diff looks weird, probably as I copied a file using IDEA and renamed it. But this is a new file.